profiles/common/modules/features/multisite_wysiwyg/multisite_wysiwyg.module

```diff
@@ -17,6 +17,21 @@ function multisite_wysiwyg_wysiwyg_editor_settings_alter(&$settings, $context) {
 
     // Provide additional custom settings.
     $settings['customConfig'] = base_path() . drupal_get_path('module', 'multisite_wysiwyg') . '/multisite_wysiwyg_config.js';
+
+    $default_theme = variable_get('theme_default', '');
+    // If a specific css file has been created for CKEditor in the default
+    // theme, we add it to the editor settings; even if it is displayed in
+    // the admin theme.
+    // This approach allows a "soft" configuration of the platform.
+    $css_file_paths = _multisite_wysiwyg_get_ckeditor_css_paths($default_theme);
+    if ($css_file_paths) {
+      foreach ($css_file_paths as $css_file_path) {
+        $settings['contentsCss'][] = $base_url . '/' . $css_file_path;
+      }
+    }
+    // This namespace must be applied even if the node edit forms are displayed
+    // with the admin theme.
+    $settings['bodyClass'] = 'ecl-editor';
```

Comment:
@voidtek,

The "ecl-editor" **must** be included here if we want to add it in the field form widget in **all** situations.

Do not forget that on some site, the content form is displayed with the admin theme (seven).
If I do that in the theme, the class will not be inserted for these sites and the magic will not work.

As its presence in the widget has no consequences if another theme is used, the dependence on theme is very weak.